README.md

```diff
@@ -5,15 +5,13 @@
 
 ### Requirements
 
-- `rubygems`
+- Ruby's [`bundler`](https://bundler.io/)
```

Comment:
```suggestion
- [Ruby](https://www.ruby-lang.org/en/downloads/)
```

Probably this, the install instructions on bundler.io just defer to that URL as well.